prov/efa/src/rxr/rxr_pkt_type_req.h

```diff
@@ -436,6 +436,24 @@ struct rxr_eager_rtw_hdr {
 	struct fi_rma_iov rma_iov[0];
 };
 
+struct rxr_dc_eager_rtw_hdr {
+	uint8_t type;
+	uint8_t version;
+	uint16_t flags;
+	/* end of rxr_base_hdr */
+	uint32_t rma_iov_count;
+	/* end of rxr_rtw_base_hdr */
+	uint32_t tx_id;
+	uint32_t padding;
+	struct fi_rma_iov rma_iov[0];
```

Comment:
I'd much rather we serialize/deserialize this on our own and not use structures that could change in the wire protocol.

Reply:
Did not quite understand.

Reply:
`struct fi_rma_iov` could change in the future (albeit unlikely). If that structure changes then our wire protocol will break.

Reply:
Introduce a new data type ```efa_rma_iov``` to replace ```fi_rma_iov`` here.

Reply:
Shall I change the type of `rma_iov` field of the `rxr_eager_rtw_hdr` too? I think `rxr_eager_rtw_hdr` and `rxr_dc_eager_rtw_hdr` are supposed to be the same.